system/classes/KO7/Request.php

```diff
@@ -990,7 +990,7 @@ public function method($method = NULL)
 		$method = strtoupper($method);
 
 		// Allow overriding method
-		$override = isset($_SERVER['HTTP_X_HTTP_METHOD_OVERRIDE']);
+		$override = $this->headers('HTTP_X_HTTP_METHOD_OVERRIDE');
```

Comment:
i think name changed to `X-Http-Method-Override`

Reply:
yes it changed

Reply:
@toitzi see https://koseven.ga/documentation/api/HTTP#request_headers